package.json

```diff
@@ -146,7 +146,10 @@
   ],
   "codegenConfig": {
     "name": "rngesturehandler_codegen",
```

Comment:
I don't like this name tbh. We have a convention of rather `rngesturehandler`

Reply:
IIRC there were conflicts with `rngesturehandler` name, hence the `_codegen` suffix.